bindings/rust/src/bindings/mod.rs

```diff
@@ -70,6 +42,11 @@ pub struct KZGProof {
     bytes: [u8; BYTES_PER_PROOF],
 }
 
+#[repr(C)]
```

Comment:
Sorry just another comment, why does this need to be `#[repr(C)]`

Reply:
I don't think it does. I'll remove.